spree_taxjar.gemspec

```diff
@@ -14,24 +14,14 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  spree_version = '>= 3.2.0', '<= 4.2.0'
+  spree_version = '>= 3.2.0'
 
   s.add_dependency 'spree_backend', spree_version
   s.add_dependency 'taxjar-ruby', '>= 2.6', '< 3.1'
+  s.add_dependency 'deface', '~> 1.0'
```

Comment:
@Vegann we also need to require `deface` in https://github.com/spree-contrib/spree_taxjar/blob/master/lib/spree_taxjar.rb